doc/governance/common_member_operations.rst

```diff
@@ -15,7 +15,7 @@ Once the proposal successfully completes, the new node automatically becomes par
 Updating Code Version
 ---------------------
 
-For new nodes to be able to join the network, the version of the code they run (as specified by the ``--enclave-file``) should be first trusted by the consortium of members.
+For new nodes to be able to join the network, the version of the code they run (as specified by ``enclave_file``) should be first trusted by the consortium of members.
```

Comment:
We ought to have a page that documents each field in the config, and ``enclave_file`` would ideally link to the right entry there.

Reply:
See `configuration.rst` for the new page. I haven't added links to all the fields on other pages, so I'll do it.

Reply:
What about writing a json schema and using https://sphinx-jsonschema.readthedocs.io/en/latest/directive.html#an-example?

Reply:
We would get links like :ref:`directive.rst#/definitions/person`, with the checking etc.

Reply:
Follow up in https://github.com/microsoft/CCF/issues/2612#issuecomment-973067527